client/router.js

```diff
@@ -121,6 +117,12 @@ Router.prototype.path = function(pathDef, fields, queryParams) {
     return fields[key] || "";
   });
 
+  path = path.replace(/\/\/+/g, "/"); // Replace multiple slashes with single slash
```

Comment:
So, this only look for two '//`. Shall we write a one to deal with any amount of slashes?